services/motion/client.go

```diff
@@ -5,6 +5,8 @@ import (
 	"context"
 
 	"github.com/edaniels/golog"
+	geo "github.com/kellydunn/golang-geo"
```

Comment:
[q] should testing for move on globe be added to `client_test.go`?

Reply:
Planned.

Reply:
It's better to add the tests in the same PR that adds the code. That way we can test handling of NaN, etc, while it's fresh in our minds.